test/functional/test_runner.py

```diff
@@ -320,6 +320,7 @@
     'feature_unsupported_utxo_db.py',
     'feature_logging.py',
     'feature_anchors.py',
+    'feature_datacarrier.py',
```

Comment:
I'd maybe suggest naming this mempool_datacarrier.py.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/25792/commits/10bfe1204fed71ff10d6bbac90912804894d454b.